metadata-service/war/src/main/resources/boot/retention.yaml

```diff
@@ -1,9 +1,15 @@
-- entity: "*"
-  aspect: "*"
+- entity: "dataHubExecutionRequest"
+  aspect: "dataHubExecutionRequestResult"
   config:
     retention:
       version:
         maxVersions: 20
```

Comment:
even 20 is a lot just for execution results, we should be fine with a lower number like 5?

Reply:
nice that sounds reasonable to me

Reply:
+1